codebase/superdarn/src.bin/tk/tool/make_raw.1.8/make_raw.c

```diff
@@ -214,11 +221,30 @@ int main (int argc,char *argv[]) {
     exit(-1);
   }
 
+  command[0]=0;
+  n=0;
+  for (int c=0; i<argc; i++) {
```

Comment:
Silly mistake that crept in from #320 

```suggestion
  for (int c=0; c<argc; c++) {
```